Cabal/src/Distribution/Simple/GHC.hs

```diff
@@ -808,213 +808,75 @@ buildOrReplLib mReplFlags verbosity numJobs pkg_descr lbi lib clbi = do
                 else do vanilla; shared
       whenProfLib (runGhcProg profOpts)
 
+  let
+    buildExtraSources mkSrcOpts wantDyn srcs =
+      sequence_
+        [ do
```

Comment:
I find the `sequence_` + list-comprehension odd. Could we use:

```suggestion
    buildExtraSources mkSrcOpts wantDyn srcs = traverse_ (buildExtraSource mkSrcOpts wantDyn) srcs
    buildExtraSource  mkSrcOpts wantDyn filename = do
```

Reply:
Ah yes that is more readable than the list comprehension